incubation.md

```diff
@@ -0,0 +1,79 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which new Jupyter Subprojects are created and evaluated for inclusion in the main Jupyter organization. In this context a Subproject is a GitHub repository with a development team that follows the Jupyter governance and contribution model. The goals of the incubation process are:
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while complying with corporate contribution restrictions.
+* Contributors can work with the community and gather feedback that will help them develop and refine a clear and concise integration proposal.
+* The broader user community can easily distinguish between stable core project feature functions and what is on the docket for potential future integration.
+* Enables the core development team to assess the readiness an applicability of proposed technologies against a backdrop of the broader project design efforts. 
+* Allows the core team to re-evaluate existing jupyter org repos to determine if any need to be moved to jupyter-incubator.
```

Comment:
I'm not sure what this bullet is for - would we really kick out existing Jupyter projects?